### DISCUSSION TITLE
Automatically hidden Windows taskbar in Win11 does not appear from within ReBoRN

### DISCUSSION THREAD

**Discussion by Groxxxy** [TT-ReBORN/Georgia-ReBORN#40] [Q&A] [answered]
Under Windows 11 a hidden taskbar does not appear when the cursor is brought into its area. Only way to access taskbar is to constantly minimise Foobar or press Windows key.

Can it be possible that from within Foobar the taskbar is accessible and appears over the application when desired.

Thanks.
![N9FD5NebBC](https://user-images.githubusercontent.com/14025275/146039089-10171725-9224-4d58-a962-e7d23d41e3e1.gif)

**Comment by TT-ReBORN** [comment, marked as answer]
Actually this happens also in Win10 with hidden taskbar IF you go either maximize or fullscreen.
I don't think I can do anything on my side, guess it's a limitation by ui_hacks and it is not supported.
TheQwertiest wants to port all ui_hacks funcs to SMP in the future, maybe it will work then...

**Reply by TT-ReBORN** [reply]
To further clarify, this happens only in maximize mode with auto-hidden taskbar.
In fullscreen mode there is no way to show the taskbar since it is fullscreen ( takes the whole screen ) and behaves like
other players ( youtube, vimeo etc. ).
